### PR TITLE
Improve-test-coverage - AuditEventConverter, JdlMetadataService, YoRC…

### PR DESCRIPTION
**Pull request by Strat1987**
Related to https://github.com/jhipster/jhipster-online/issues/213

### CODE REVIEW

**Review by SudharakaP — CHANGES_REQUESTED** [review]
@Strat1987 : Hi, thanks for submitting the PR. 😄 

1) Could you please sign the CLA. 

2) There seems to be a CI failure in the newly added file `AuditEventConverterTest.java` at line 59. I think there's a problem with the syntax. 

```

-> at io.github.jhipster.online.config.audit.AuditEventConverterTest.convertToAuditEventNullEvent(AuditEventConverterTest.java:59)

You cannot use argument matchers outside of verification or stubbing.
Examples of correct usage of argument matchers:
    when(mock.get(anyInt())).thenReturn(null);
    doThrow(new RuntimeException()).when(mock).someVoidMethod(anyObject());
    verify(mock).someMethod(contains("foo"))
```

**Comment by Strat1987** [comment]
1. the version wouldn't load last time for the CLA, was successful this time around.
2. Using IntelliJ test runner, this assertion does succeed, I do feel the implementation is strange since it adds null objects to the list. not empty check suffices as I'm only adding the null value, no need to explicitly assert it

**Comment by SudharakaP** [comment]
@Strat1987 : The CLA doesn't seem to be signed yet. Did you already signed it. Maybe there's something wrong on our side? 🤔 

cc: @pascalgrimaud

File: src/main/java/io/github/jhipster/online/service/JdlMetadataService.java [hunk 1]
```diff
@@ -148,7 +148,7 @@ public void updateJdlContent(JdlMetadata jdlMetadata, String content) throws Fil
         } else if (jdlMetadata.isPresent() && jdlMetadata.get().getUser().equals(this.getUser())) {
             return jdlMetadata;
         } else {
-            throw new AccessDeniedException("Current user does not have access to this JDL file");
```
Comment (SudharakaP) [inline src/main/java/io/github/jhipster/online/service/JdlMetadataService.java:151]: I think this change should be reverted. The alignment seems removed here. 🤔

**Comment by pascalgrimaud** [comment]
I think it's because the commits come from @roexber account, and you tried to sign the cla with @Strat1987 account

File: src/main/java/io/github/jhipster/online/service/JdlMetadataService.java (hunk 1 above)
Comment (pascalgrimaud) [reply src/main/java/io/github/jhipster/online/service/JdlMetadataService.java:151]: we should add prettier-java to solve this kind of formatting

**Comment by SudharakaP** [comment]
> I think it's because the commits come from @roexber account, and you tried to sign the cla with @Strat1987 account

@pascalgrimaud : Thanks. Nice catch. 👍🏼

File: src/main/java/io/github/jhipster/online/service/JdlMetadataService.java (hunk 1 above)
Comment (SudharakaP) [reply src/main/java/io/github/jhipster/online/service/JdlMetadataService.java:151]: @pascalgrimaud : Good idea; I've added it at https://github.com/jhipster/jhipster-online/pull/249.

**Comment by roexber** [comment]
I've now accepted CLA using the github account I also used for the commits, apologies

**Comment by pascalgrimaud** [comment]
No worry @roexber : don't be sorry for helping Open Source projects :-)

**Comment by SudharakaP** [comment]
> I've now accepted CLA using the github account I also used for the commits, apologies

@Strat1987 : No worries, thanks a bunch. Can you re-base and run prettier; `npm run prettier:format` so that we have the formatting consistent. 😄

**Comment by roexber** [comment]
@SudharakaP Done, It's a challenge for me performing this many history changing GIT commands which I'm really not accustomed with in my day to day.

**Review by SudharakaP — APPROVED** [review]

**Comment by SudharakaP** [comment]
@roexber : Thanks much for your contribution. Looks awesome. 👍🏼

**Comment by roexber** [comment]
@SudharakaP thanks for the kind words. I'm happy to contribute since JHipster greatly helped in coming back to the spring eco system

**Comment by SudharakaP** [comment]
You are welcome. 🙃

**Pull request merged**